### PR TITLE
Added Service Mesh Focussed Section

### LINKED ISSUE #1426: Implementation of the Service mesh focused section on homepage
**Current Behavior**
We currently have no implementation of the Service mesh focused section on homepage 

**Desired Situation**
The design of the home page is currently approved on figma, and we will appreciate someone jumping on the task of implementing it 

https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=1869%3A2967

Checkout the screenshot of design below for easy navigation and identification
focused

[Optional] **Alternatives**

[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by chinmaym07**
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

**Description**

This PR fixes #1426 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/ServiceMesh/home/service-mesh-focussed.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        flex-direction:column;
+        justify-content:center;
+        align-items:center;
+        padding:20px 10px;
+    }
+    .head-cont-2 h1{
+        display:flex;
+        flex-direction:column;
+        padding:15px;
+        margin:10px;
```
Comment (Nikhil-Ladha) [inline src/sections/ServiceMesh/home/service-mesh-focussed.js:22]: This margin can be removed.

File: src/sections/ServiceMesh/home/service-mesh-focussed.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    flex-direction:column;
+    justify-content:center;
+    align-items:center;
+   } 
+    
+    .head-cont-2{
+        display:flex;
+        flex-direction:column;
+        justify-content:center;
+        align-items:center;
```
Comment (Nikhil-Ladha) [inline src/sections/ServiceMesh/home/service-mesh-focussed.js:15]: None of this is required, just add `text-align: center` that would do.

File: src/sections/ServiceMesh/home/service-mesh-focussed.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        padding:20px 10px;
+    }
+    .head-cont-2 h1{
+        display:flex;
+        flex-direction:column;
```
Comment (Nikhil-Ladha) [inline src/sections/ServiceMesh/home/service-mesh-focussed.js:20]: Same as above.
Instead os using `span` within `h1` use 2 `h1` tags. It should work.

File: src/sections/ServiceMesh/home/service-mesh-focussed.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+        }
+        
+    }
+    @media only screen and (max-width:500px){
+
+    .card-cont-1{
+        width:93%;
+        margin:10px;
+        flex-direction:column;
+        justify-content:center;align-items:center;
```
Comment (Nikhil-Ladha) [inline src/sections/ServiceMesh/home/service-mesh-focussed.js:103]: Break to next line.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Use `rem` unit instead of `px` wherever possible. No need to do exact calculation like `6.125rem` or `7.9265rem` something like that, just round-off to nearest number like `9.2625rem` to `9.25 or 9.3` whichever works.

File: src/sections/ServiceMesh/home/service-mesh-focussed.js (hunk 4 above)
Comment (chinmaym07) [reply src/sections/ServiceMesh/home/service-mesh-focussed.js]: Fixed! .Thanks for the Review.! @Nikhil-Ladha 👍🏻 😀

**Comment by Nikhil-Ladha** [comment]
Are you planning to update `px` to `rem`?

**Comment by chinmaym07** [comment]
> Are you planning to update `px` to `rem`?

Should I change everywhere , wherever I have used px .?

File: src/sections/ServiceMesh/home/index.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+
+import React from "react";
+import MeshFocusWrapper from "./service-mesh-focussed";
+//import { Row, Col } from "../../../reusecore/Layout";
+import BlockQouteImage from "../../../assets/images/blockqoute/quote-left.svg";
+import Button from "../../../reusecore/Button/index";
```
Comment (Nikhil-Ladha) [inline src/sections/ServiceMesh/home/index.js:6]: Mentioning `index` in the path is not required. That's the default setting.

File: src/sections/ServiceMesh/home/index.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
+
+import React from "react";
+import MeshFocusWrapper from "./service-mesh-focussed";
+//import { Row, Col } from "../../../reusecore/Layout";
```
Comment (Nikhil-Ladha) [inline src/sections/ServiceMesh/home/index.js:4]: Remove unused imports.

File: src/sections/ServiceMesh/home/index.js [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+          <div className="content-col">
+            <img src={BlockQouteImage} alt="Qoute-left"/>
+            <p className="content">
+              <strong>
+              "Diverse Microservices patterns and technologies, together with the requirements of given microservice applications,provide myriad opportunities 
+              for service mesh differentiation and specialization - including meshes native to specific cloud platforms. This will lead to a world where many 
+              enterprises use multiple service mesh products, whether separately or together."
+              </strong>
+            </p>
+            <Button>
```
Comment (Nikhil-Ladha) [inline src/sections/ServiceMesh/home/index.js:32]: Add the link to this book, `/books/the-enterprise-path-to-service-mesh-architectures`.

**Comment by Nikhil-Ladha** [comment]
> > Are you planning to update `px` to `rem`?
> 
> Should I change everywhere , wherever I have used px .?

Yup. Very small values like 1px to 7px can be ignored.

File: src/sections/ServiceMesh/home/index.js (hunk 7 above)
Comment (chinmaym07) [reply src/sections/ServiceMesh/home/index.js]: Fixed !

**Review by Nikhil-Ladha — APPROVED** [review]
Looking good....Thanks, @chinmaym07 👍🏼

**Pull request merged**